odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/model/SaleOrder.java

```diff
@@ -37,4 +37,7 @@ public class SaleOrder implements OdooResource {
 
     @JsonProperty("type_name")
     private String orderTypeName;
+
+    @JsonProperty("x_customer_weight")
```

Comment:
Can this be passed as an environment or runtime property to allow for configurability?

Reply:
I feel like its better not to make this configurable as this is just a field name.

Reply:
Implementers may want to name this field differently, which could make customization more challenging. Since this is a custom field, it makes sense to leave it as a configurable option with a default value.

Reply:
For naming the field differently we will need to change the `odoo_initializer` config files as well. I don't think it is possible to configure the `odoo_initializer` that way. CMIIW @Ruhanga